install/servlet/bin/ts.jte

```diff
@@ -192,7 +192,7 @@ env.ts_win32.menu=true
 # The sigTestClasspath specify the location of servlet implementation 
 # classes that used for signature test
 ##########################################################################
-sigTestClasspath=${java.home}/lib/rt.jar${pathsep}${servlet.classes}
```

Comment:
@markt-asf sorry that I missed this before, but I have a question.  Does install/servlet/bin/ts.jte.jdk11 also need a change?  Currently it has:
```
sigTestClasspath=${java.home}/lib/rt.jar${pathsep}${servlet.classes}
```

Reply:
The approach of having a separate `ts.jte` file for Java 11 post-dates this PR. Strictly you need `${java.home}/lib/rt.jar${pathsep}${servlet.classes}` for Java 8 and `${java.home}/lib/modules${pathsep}${servlet.classes}` for Java 11. The new version I provided in this PR enables you to use the same file for both Java 8 and Java 11.

Personally, I'd prefer to avoid the use of a Java version specific `ts.jte` file but that isn't my call to make. Happy to adjust this PR however you see fit.

Reply:
@markt-asf for some ts.jte files it is easier to have two separate files.

Lets merge this change as is and figure out separately if its possible to have one ts.jte for Servlet versus two.

Reply:
CC @alwin-joseph ^